libs/ui/src/lib/datepicker/datepicker.component.html

```diff
@@ -0,0 +1,39 @@
+<div
+  class="cvi-datepicker__input-container"
+  [ngClass]="{ 'cvi-datepicker__input-container--focused': focus }"
+>
+  <label id="datePickerLabel" [attr.for]="htmlId" class="cvi-screenreader-text"
+         *ngIf="screenreaderLabel$ | async as label">{{label}}</label>
+  <span id="dateDescription" class="cvi-screenreader-text"
```

Comment:
The ids should be unique. This id will likely stop being unique after a second datepicker appears on the same page.

In this repo this problem is resolved in a similar component ([`tab-group.component.html`](https://github.com/e-gov/cvi/blob/main/libs/ui/src/lib/tabs/tab-group.component.html)) with [uuidv4](https://www.npmjs.com/package/uuidv4) package.

This problem applies to other ids in this component as well.